RELEASE-NOTES.md

```diff
@@ -1,5 +1,85 @@
+- Fix typing on `GlobalConcurrencyLimitUpdate` — https://github.com/PrefectHQ/prefect/pull/14005
+- Bump `TaskRunWaiter` cache size and expiration time and pin task worker thread pool workers to passed `limit` — https://github.com/PrefectHQ/prefect/pull/14030
+- Fix Kubernetes guide to avoid suggesting you need cloud — https://github.com/PrefectHQ/prefect/pull/14040
+- Require a `PREFECT_API_URL` to start a worker — https://github.com/PrefectHQ/prefect/pull/13942
+- Remove `LiteralResult` and handle singleton values the same — https://github.com/PrefectHQ/prefect/pull/13905
+- Resolve an infinite loop distributing events to a websocket subscriber — https://github.com/PrefectHQ/prefect/pull/13877
+
+### Documentation
+- Remove gerunds and redundant prefixes from CLI command help text — https://github.com/PrefectHQ/prefect/pull/13788
+- Remove script tag — https://github.com/PrefectHQ/prefect/pull/13852
+- Quickstart 3.0rc polish — https://github.com/PrefectHQ/prefect/pull/13879
+- Update `pydantic` syntax in docs — https://github.com/PrefectHQ/prefect/pull/13888
+- Improves integrations docs formatting  — https://github.com/PrefectHQ/prefect/pull/13934
+- Simplifies integration docs  — https://github.com/PrefectHQ/prefect/pull/13926
+- Fix code block formatting in automations concepts docs — https://github.com/PrefectHQ/prefect/pull/13959
+- Add react docs — https://github.com/PrefectHQ/prefect/pull/13965
+- Update Quickstart install instructions for 3.0 — https://github.com/PrefectHQ/prefect/pull/13988
+- Fix docs formatting in integrations page — https://github.com/PrefectHQ/prefect/pull/13999
+- Removed references to SequentialTaskRunner — https://github.com/PrefectHQ/prefect/pull/13996
+- Removes extraneous docs files — https://github.com/PrefectHQ/prefect/pull/14016
+- Remove extra arrow in top button — https://github.com/PrefectHQ/prefect/pull/14052
+- Adds cards showing all integration libraries to the integrations page — https://github.com/PrefectHQ/prefect/pull/14051
+- Update contribute section of the docs — https://github.com/PrefectHQ/prefect/pull/14055
+- Add documentation about supported function types — https://github.com/PrefectHQ/prefect/pull/14060
+
+### Breaking Changes
+- Remove unused attributes from `EngineContext` — https://github.com/PrefectHQ/prefect/pull/13917
+
+### Integrations
+
+#### prefect-dbt
+
+- Add back logging of dbt commands — https://github.com/PrefectHQ/prefect/pull/13853
+
+#### prefect-databricks
+
+- Support job parameters for Databricks jobs — https://github.com/PrefectHQ/prefect/pull/13642
+- Restore `jobs_runs_submit_by_id_and_wait_for_completion` — https://github.com/PrefectHQ/prefect/pull/13958
+
+## Contributors
+- @LarryUllman
+- @Waiwait
+- @padbk
+- @stellardave
+
+**All changes**: https://github.com/PrefectHQ/prefect/compare/3.0.0rc2...3.0.0rc3
+
 ## Release 3.0.0rc1
```

Comment:
did we not have an entry for rc2?

Reply:
I didn't see one. We can add it back in later if we want to.